yugabyte/src/yugabyte/append.clj

```diff
@@ -9,11 +9,29 @@
             [jepsen.tests.cycle :as cycle]
             [jepsen.tests.cycle.append :as append]))
 
+(defn workload-rr
+  [opts]
+  (-> (append/test {:key-count          32
+                    :max-txn-length     4
+                    :max-writes-per-key 1024
+                    :anomalies          [:G1 :G2-item]
+                    :consistency-models [:repeatable-read]
+                    :additional-graphs  [elle/realtime-graph]})))
+
+(defn workload-rc
+  [opts]
+  (-> (append/test {:key-count          32
+                    :max-txn-length     4
+                    :max-writes-per-key 1024
+                    :anomalies          [:G1]
+                    :consistency-models [:read-committed]
+                    :additional-graphs  [elle/realtime-graph]})))
+
 (defn workload
```

Comment:
`workload-serializable`
Also, maybe make `workload` a private reusable function since almost all config options are the same?

Reply:
Fixed